substratest/factory.py

```diff
@@ -221,11 +222,24 @@ class DataSampleSpec(_Spec):
     test_only: bool
     data_manager_keys: typing.List[str]
 
-    def move_data_to(self, destination):
-        destination = destination if destination.endswith('/') else destination + '/'
-        destination = tempfile.mkdtemp(dir=destination)
-        shutil.move(self.path, destination)
-        self.path = os.path.join(destination, os.path.basename(self.path))
+    def move_data_to(self, copy_destination, minikube=False):
```

Comment:
Why did you rename `destination` to `copy_destination`? I think `copy_` is misleading (the function is called `move_` and redundant with the function name.
A better name would be `destination_folder`? It would add more information to the code reader.

What do you think?

Reply:
We should maybe rename this function to `move_data_to_server()` as it is less generic than before. And it would help to understand the meaning of the minikube option.

Reply:
Yes, good points !